pkg/azuremachine/validate_create.go

```diff
@@ -59,7 +59,14 @@ func (a *CreateValidator) Validate(ctx context.Context, request *v1beta1.Admissi
 		return microerror.Maskf(errors.ParsingFailedError, "unable to parse AzureMachine CR: %v", err)
 	}
 
-	err := generic.ValidateOrganizationLabelContainsExistingOrganization(ctx, a.ctrlClient, cr)
+	err := cr.ValidateCreate()
+	if CAPZIsSSHFieldValidationError(err) {
```

Comment:
This validation error came up when using an empty SSH field, which is something we actually require to be empty. Let's ignore it for now.